meta-hpe/meta-gxp/recipes-connectivity/virtualnic/files/LICENSE

```diff
@@ -0,0 +1,204 @@
+
```

Comment:
where did /recipes-connectivity/virtualnic come from?

Reply:
It is new code